src/components/Dashboard/ManageUsersPage/AddUserPage.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import 'components/Dashboard/ManageUsersPage/AddUserPage.css';
+import EditUser from 'components/Dashboard/ManageUsersPage/EditUser';
+import { Redirect } from 'react-router-dom';
+import gql from 'graphql-tag';
+import { graphql } from 'react-apollo';
+import generator from 'generate-password';
+import AddUserModal from 'components/Dashboard/ManageUsersPage/AddUserModal';
```

Comment:
Don't need to change this now, but grouping external and internal imports can make things easier to read.

E.g. 

```
import React, { Component } from 'react';
import { Redirect } from 'react-router-dom';
import gql from 'graphql-tag';
import { graphql } from 'react-apollo';
import generator from 'generate-password';

import EditUser from 'components/Dashboard/ManageUsersPage/EditUser';
import AddUserModal from 'components/Dashboard/ManageUsersPage/AddUserModal';
import 'components/Dashboard/ManageUsersPage/AddUserPage.css';
```